src/operon-runtime/deploy.ts

```diff
@@ -5,6 +5,7 @@ import FormData from 'form-data';
 
 export async function deploy(appName: string, host: string) {
 
+    const tempHardcodedToken = "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwidXNlcm5hbWUiOiJKb2huIERvZSIsImlhdCI6MTUxNjIzOTAyMn0.p5Csu2THYW5zJys2CWdbGM8GaWjpY6lOQpdLoP4D7V4";
```

Comment:
A placeholder while we build a working authentication system for the cloud prototype.